logger/logger.go

```diff
@@ -172,6 +172,6 @@ func debugEnabledOnKernelCmdline() bool {
 	if err != nil {
 		return false
 	}
-	l := strings.Split(string(buf), " ")
+	l := strings.Fields(string(buf))
```

Comment:
could we use `l, _ := strutil.KernelCommandLineSplit(strings.TrimSpace(buf))` ?

Reply:
This looks like the better option indeed.

Reply:
Opening PR #9495 to change as suggested.